src/app/bisoncoin/src/components/Button.vue

```diff
@@ -24,7 +29,8 @@ export default class Button extends Vue {
   transition: 0.2s linear;
   margin-top: 100px;
   margin-bottom: 50px;
-  border: 1px solid black;
+  border: 1px solid $border-color;
+  color: $button-text;
 }
 
 .button:hover {
```

Comment:
Or at least move "rgba(0,0,0,0.2)" to a variable since we reuse it down here for box-shadow